drop-down.d.ts

```diff
@@ -54,6 +54,7 @@ export  class ValueList<T> extends ObservableArray<ValueItem<T>> implements Item
     public getDisplay(index: number): string;
     public getValue(index: number): T;
     public getIndex(value: T): number;
+    constructor(items: ValueItem<T>[]);
```

Comment:
Should be an option parameter.

Reply:
Good catch, I added a second parameterless constructor. I hope that is alright.  Thanks for the excellent plugin dude.